cypress/e2e/basicTests.cy.ts

```diff
@@ -0,0 +1,106 @@
+/// <reference types="cypress" />
+
+//Admin empty export to make this spec file a module
+export { }
+
+const restaurantPlacardSelector = 'li.place-card__list-item'
+const dropdownSelector = '.location-dropdown__button-text'
+const dropdownOptionsSelector = 'li button.location-dropdown__list-button'
+const footerSelector = '.footer'
+const placeCardCitySelector = '.place-card__city'
+const linkOfRestaurantSelector = '.place-card__name-link'
+
+describe('some basic sanity tests', () => {
```

Comment:
Please take a look at the #407 for an example of how I would like to consistently structure the end-to-end tests. They are structured as acceptance criteria so that so that the specifications for Sparkeats can be directly mapped to the tests.

Let me know if you have questions or if this doesn't make sense!

Also, if this feels like more work than you expected, I would suggest only writing one or two tests, or only including the installation and setup in this PR.

Reply:
Sure let me give it one more shot. :)